Core/ALife.Core/Scenarios/TestScenarios/RectangleScenario.cs

```diff
@@ -88,15 +90,15 @@ public virtual void CollisionBehaviour(Agent me, List<WorldObject> collisions)
 
         public virtual void PlanetSetup()
         {
-            Zone nullZone = new Zone("Null", "random", System.Drawing.Color.Black, new Geometry.Shapes.Point(0, 0), 1000, 1000);
+            Zone nullZone = new Zone("Null", "random", Colour.Black, new Geometry.Shapes.Point(0, 0), 1000, 1000);
             Planet.World.AddZone(nullZone);
 
             //Geometry.Shapes.Point ap = new Geometry.Shapes.Point(30, 30);
-            //Agent a = AgentFactory.CreateAgent("Agent", nullZone, nullZone, System.Drawing.Color.Red, 0);
+            //Agent a = AgentFactory.CreateAgent("Agent", nullZone, nullZone, System.Drawing.Colour.Red, 0);
```

Comment:
This commented out code should be updated.

Reply:
Will do once I'm home!